3rdparty/CMakeLists.txt

```diff
@@ -222,30 +218,44 @@ ADD_SUBDIRECTORY(toml11 EXCLUDE_FROM_ALL)
 SET(SPDLOG_ENABLE_PCH ON CACHE BOOL "" FORCE)
 # Not yet supported by github runners
 # SET(SPDLOG_USE_STD_FORMAT ON CACHE BOOL "" FORCE)
+SET(SPDLOG_BUILD_PIC  ON CACHE BOOL "" FORCE)
 ADD_SUBDIRECTORY(spdlog EXCLUDE_FROM_ALL)
 ADD_SUBDIRECTORY(longeronpp EXCLUDE_FROM_ALL)
 
 # For Windows: Prevent overriding the parent project's compiler/linker settings
 SET(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 ADD_SUBDIRECTORY(googletest EXCLUDE_FROM_ALL)
 
-SET(FT_DISABLE_ZLIB        ON CACHE BOOL "" FORCE)
-SET(FT_DISABLE_BZIP2       ON CACHE BOOL "" FORCE)
-SET(FT_DISABLE_PNG         ON CACHE BOOL "" FORCE)
-SET(FT_DISABLE_HARFBUZZ    ON CACHE BOOL "" FORCE)
-SET(FT_DISABLE_BROTLI      ON CACHE BOOL "" FORCE)
-SET(SKIP_INSTALL_ALL       ON CACHE BOOL "" FORCE)
-SET(SKIP_INSTALL_LIBRARIES ON CACHE BOOL "" FORCE)
-SET(BUILD_SHARED_LIBS      ON CACHE BOOL "" FORCE)
-ADD_SUBDIRECTORY(freetype EXCLUDE_FROM_ALL)
-
-INSTALL(TARGETS freetype
-        EXPORT RmlUiTargets
-        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-       )
+# SET(FT_DISABLE_ZLIB        ON CACHE BOOL "" FORCE)
+# SET(FT_DISABLE_BZIP2       ON CACHE BOOL "" FORCE)
+# SET(FT_DISABLE_PNG         ON CACHE BOOL "" FORCE)
+# SET(FT_DISABLE_HARFBUZZ    ON CACHE BOOL "" FORCE)
+# SET(FT_DISABLE_BROTLI      ON CACHE BOOL "" FORCE)
+# SET(SKIP_INSTALL_ALL       ON CACHE BOOL "" FORCE)
+# SET(SKIP_INSTALL_LIBRARIES ON CACHE BOOL "" FORCE)
+# SET(BUILD_SHARED_LIBS      OFF CACHE BOOL "" FORCE)
+# ADD_SUBDIRECTORY(freetype EXCLUDE_FROM_ALL)
```

Comment:
if i recall correctly, we only need freetype for RmlUi.

So if we're going to go full steam on godot, then we can just drop RmlUi and freetype.

but in general, best not to leave commented out code in the source files.